modules/admin_manual/pages/configuration/server/email_configuration.adoc

```diff
@@ -12,29 +12,21 @@ ownCloud is capable of sending emails for a range of reasons. These include:
 * Changes in files
 * Activity notifications
 
-To make use of them, users need to configure which notifications they
-want to receive. They can do this on their Personal pages.
+To make use of them, users need to configure which notifications they want to receive. They can do this on their Personal pages.
 
-NOTE: To be able to send emails, a functioning mail server must be available, 
-whether locally in your network, or remotely.
+NOTE: To be able to send emails, a functioning mail server must be available, whether locally in your network, or remotely.
 
-== Configuring an SMTP Server
+== The Graphical Email Configuration Wizard
 
-To configure ownCloud to interact with an SMTP server, you can either
-update `config/config.php` by hand, or use the graphical Email
-Configuration Wizard, which updates `config/config.php` for you.
+The wizard supports two mail server types: _SMTP_ and _PHP_. Use SMTP for a remote email server, and PHP when your mail server is on the same machine as ownCloud.
 
-=== The Graphical Email Configuration Wizard
+In most cases the `smtp` option is best, because it removes the extra step of passing through PHP, and you can control all of your mail server options in one place, in your ownClouds email server configuration.
```

Comment:
```suggestion
In most cases the `smtp` option is best, because it removes the extra step of passing through PHP, and you can control all of your mail server options in one place, in your ownCloud's email server configuration.
```